### PR TITLE
Zero-shot image classification task guide

### PR DESCRIPTION
**Pull request by MKhalusova**
This PR adds the inference task guide for zero-shot image classification. It adds examples of inference with a pipeline and manual inference.

### CODE REVIEW

File: docs/source/en/tasks/zero_shot_image_classification.mdx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+Zero-shot image classification is a task that involves classifying images into different categories using a model that was
+not explicitly trained on data containing labeled examples from those specific categories.
+
+Traditionally, image classification requires training a model on a specific set of labeled images, and this model learns to
+"map" certain image features to labels. When there's a need to use such model for a classification task that introduces a
+new set of labels, fine-tuning is required to "recalibrate" the model.
+
+In contrast, a pre-trained model that can be used for zero-shot image classification typically has been trained on a large
+dataset of images and associated descriptions. Such model has learned aligned vision-language representations. These can
+be used for many downstream tasks including zero-shot image classification.
```
Comment (alaradirik) [inline docs/source/en/tasks/zero_shot_image_classification.mdx:26]: Just rephrasing to clarify the use of multi-modal models for zero-shot image :
```suggestion
In contrast, zero-shot or open vocabulary image classification models are typically multi-modal models that have been trained on a large
dataset of images and associated descriptions. These models learn aligned vision-language representations that can be used for many downstream tasks including zero-shot image classification.
```

File: docs/source/en/tasks/zero_shot_image_classification.mdx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+This is a more flexible approach to image classification that allows models to generalize to new and unseen categories
+without the need for additional training data.
```
Comment (alaradirik) [inline docs/source/en/tasks/zero_shot_image_classification.mdx:29]: I think it'd be great to emphasize the use of free-form text queries as opposed to rigid labels:
```suggestion
without the need for additional training data and enables users to query images with free-form text descriptions of their target objects .
```

**Review by alaradirik — APPROVED** [review]
Thanks for adding this! I left two suggestions but LGTM otherwise

**Review by amyeroberts — APPROVED** [review]
LGTM - thanks for adding this!

**Pull request merged**